curriculum/challenges/english/25-front-end-development/quiz-javascript-problem-solving-and-algorithmic-thinking/66edd4bc0ce399f475ad6f9b.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 ### --question--
 
 #### --text--
 
-Placeholder question
+Given a Roman numeral string, convert it to its corresponding integer value. For example, `"IX"` should return `9`.
 
 #### --distractors--
 
-Placeholder distractor 1
+Parse the string from left to right and add the values of individual numerals.
 
 ---
 
-Placeholder distractor 2
+Parse each character and multiply by its index.
 
 ---
 
-Placeholder distractor 3
+Use the `parseInt()` function on the Roman numeral string.
 
 #### --answer--
 
-Placeholder answer
+Parse the string from right to left, adding the values of individual numerals and subtracting when a smaller numeral comes before a larger numeral.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How can one check if a given string is a palindrome (the same forwards and backwards), for example `"madam"`?
 
 #### --distractors--
 
-Placeholder distractor 1
+Use Slow and Fast Pointer.
 
 ---
 
-Placeholder distractor 2
+Compare it to its reverse after sorting the string.
 
 ---
 
-Placeholder distractor 3
+Use regular expressions to remove all non-alphabetic characters and check equality.
 
 #### --answer--
 
-Placeholder answer
+Compare the string to its reverse.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Given a number `n`, write a function that prints numbers from `1` to `n`. For multiples of `3`, print `“Fizz”` for multiples of `5`, print `“Buzz”` and for multiples of both `3` and `5`, print `“FizzBuzz”`. What is the simplest approach to this?
 
 #### --distractors--
 
-Placeholder distractor 1
+Use Math.random() to generate numbers between 1 and n and check the result.
 
 ---
 
-Placeholder distractor 2
+Use a for loop and check divisibility in-order of `3`, `5` & `15` with if conditions or switch.
 
 ---
 
-Placeholder distractor 3
+Create a recursive function that checks divisibility by `3` and `5`.
 
 #### --answer--
 
-Placeholder answer
+Use a for loop and check divisibility in-order of `15`, `5` & `3` with if conditions or switch.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The Caesar cipher is a simple encryption technique where each letter in a string is shifted forward by a fixed number of positions in the alphabet. How would you shift all letters in a string by `3` positions?
 
 #### --distractors--
 
-Placeholder distractor 1
+Subtract `3` from the ASCII value of each character.
 
 ---
 
-Placeholder distractor 2
+Replace each vowel with the next consonant.
 
 ---
 
-Placeholder distractor 3
+Add `3` to the ASCII value of each character, but only for uppercase letters.
 
 #### --answer--
 
-Placeholder answer
+Add `3` to the ASCII value of each character, wrapping around for `'z'` and `'Z'`.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Anagrams are words formed by rearranging letters of another word. How can one determine if two strings are anagrams?
 
 #### --distractors--
 
-Placeholder distractor 1
+Compare the character count of both strings.
```

Comment:
Why not?

Reply:
I intended `character count` to mean number of characters in string i.e. length of string. Again, trying to mislead. Let me know if it requires rewording.

Reply:
Yes it does require rewording, I interpeted as "compare how many times a letter appear in each string"